lib/apiScenario/gen/testScenarioGenerator.ts

```diff
@@ -1,4 +1,4 @@
-import { join as pathJoin, relative as pathRelative, dirname } from "path";
+// import { join as pathJoin, relative as pathRelative, dirname } from "path";
```

Comment:
Can you remove them if not referenced?

Reply:
testScenarioGenerator is to generate api scenario from recording. We just fixed the compile errors, and we will remove this or fix it next time.